src/util.rs

```diff
@@ -79,7 +68,7 @@ impl Ord for Hash {
 
 impl Display for Hash {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", encode(self.0.as_bytes()))
+        write!(f, "{}", hex::encode(self.0.as_bytes()))
```

Comment:
why is this hex?

Reply:
I can make it base32 as well, but since the hashes are never publicly visible I made it hex. Can make it base32lowernopad as well.